src/SFA.DAS.ApplyService.InternalApi/Controllers/RoatpModeratorController.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApplyService.Application.Apply.Moderator;
+using SFA.DAS.ApplyService.Domain.Apply.Moderator;
+using SFA.DAS.ApplyService.InternalApi.Services.Assessor;
+using SFA.DAS.ApplyService.InternalApi.Types.Assessor;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class RoatpModeratorController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly IAssessorSequenceService _sequenceService;
+        private readonly IAssessorPageService _pageService;
+        private readonly IAssessorSectorService _sectorService;
+        private readonly IAssessorSectorDetailsService _sectorDetailsService;
+
+        public RoatpModeratorController(IMediator mediator,
+            IAssessorSequenceService assessorSequenceService, IAssessorPageService assessorPageService,
+            IAssessorSectorService assessorSectorService, IAssessorSectorDetailsService assessorSectorDetailsService)
+        {
+            _mediator = mediator;
+            _sequenceService = assessorSequenceService;
+            _pageService = assessorPageService;
+            _sectorService = assessorSectorService;
+            _sectorDetailsService = assessorSectorDetailsService;
+        }
+
+        [HttpGet("Moderator/Applications/{applicationId}/Overview")]
+        public async Task<List<AssessorSequence>> GetModeratorOverview(Guid applicationId)
+        {
+            var overviewSequences = await _sequenceService.GetSequences(applicationId);
+
+            return overviewSequences.OrderBy(seq => seq.SequenceNumber).ToList();
+        }
+
+        [HttpPost("Moderator/Applications/{applicationId}/Sectors")]
+        public async Task<List<AssessorSector>> GetSectors(Guid applicationId, [FromBody] GetSectorsRequest request)
```

Comment:
Why is this a post? is the one element, userid, sensitive?

Reply:
I think so, it was copied across from Assessor which did it in the same way